docs/tutorial/truncating.md

```diff
@@ -0,0 +1,281 @@
+# Truncating
```

Comment:
No link is done with rounding ?

Reply:
I'll do that when we refactor rounding to use the new operators, right now it's a bit of a mess. I'll also explain how these operators are implemented in depth.